metadata-ingestion/src/datahub/ingestion/graph/client.py

```diff
@@ -560,11 +562,12 @@ def get_entity_semityped(self, entity_urn: str) -> AspectBag:
         something, even if the entity doesn't actually exist in DataHub.
 
         :param entity_urn: The urn of the entity
+        :param Optional[List[str]] aspects: Optional list of aspect names being requested (e.g. [schemaMetadata, datasetProperties])
```

Comment:
type info is already in the signature. sphinx can figure out how to merge it in for generated docs

```suggestion
        :param aspects: Optional list of aspect names being requested (e.g. ["schemaMetadata", "datasetProperties"])
```